tests/framework/test_time.py

```diff
@@ -34,7 +34,7 @@ def components():
     return [
         MockComponentA("gretchen", "whimsy"),
         Listener("listener"),
-        MockComponentB("spoon", "antelope", 23),
+        MockComponentB("spoon", "antelope", "23"),
```

Comment:
Why was this change needed?

Reply:
Because later we `self.name.split("_observer")` and you can't split an int.